app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)

```

Comment:
Code Style Issue: It's better to use parentheses when importing multiple classes/modules for better readability. So, your import statement could look like this:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```